client/identities.go

```diff
@@ -47,6 +48,12 @@ type LocalIdentity struct {
 	UserID *uint32 `json:"user-id" yaml:"user-id"`
 }
 
+// BasicIdentity holds identity configuration specific to the "basic" type
+// (for HTTP basic authentication).
```

Comment:
This doc comment uses the word 'basic' quite a lot, is there anywhere in this repo where we explain what 'basic' means? I think it could be good to add an explanation of what it entails, or at least link to somewhere which does this.

Also should it be _http basic_ authentication or _basic http_ authentication?

Reply:
Thanks for pointing this out, I will add more comments in the code where it mentions basic type identities.

Regarding the name, I do agree "HTTP basic" is better than simply "basic", but the name name "basic" has been decided in the spec review meeting with Gustavo, so I will keep it as it is for now.